src/test/scala/chiselTests/StrongEnum.scala

```diff
@@ -555,6 +576,12 @@ class StrongEnumSpec extends ChiselFlatSpec with Utils {
   it should "correctly check if the enumeration is one of the values in a given sequence" in {
     assertTesterPasses(new IsOneOfTester)
   }
+
+  it should "work with Printables" in {
+    ChiselStage.emitChirrtl(new LoadStoreExample) should include(
+      """printf(clock, UInt<1>("h1"), "%c%c%c%c%c", EnumRecord[0], EnumRecord[1], EnumRecord[2], EnumRecord[3], EnumRecord[4])"""
+    )
```

Comment:
I don't love this check because it's very specific to implementation details and doesn't really capture the functionality of what's happening. I'd rather see an execution driven test that walks through all of the concrete enum values (and even an invalid state!) and checks that the correct thing is printed at simulation time.